src/components/modebar/buttons.js

```diff
@@ -47,19 +47,26 @@ var modeBarButtons = module.exports = {};
 
 modeBarButtons.toImage = {
     name: 'toImage',
-    title: function(gd) { return _(gd, 'Download plot as a png'); },
+    title: function(gd) { return _(gd, 'Download plot'); },
```

Comment:
We can't make dynamic strings here, so `_(gd, 'Download plot as a ' + format)` won't work, but perhaps we can keep `'Download plot as a png'` as the default string and only degrade to `'Download plot'` if it's not a png? Two reasons:
- I don't want to unnecessarily break all our existing translations, for folks who are *not* using this new option.
- I do find "as a png" helpful, to clarify that it's not the data or json for example, though I guess the icon mitigates that substantially.